### PR TITLE
Add userHomeDir

### PR DESCRIPTION
**Pull request by ekaragodin**

### CODE REVIEW

File: os/mod.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+// Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
+
+/**
+ * Returns the current user's home directory.
+ * On Unix, including macOS, it returns the $HOME environment variable.
+ * On Windows, it returns %USERPROFILE%.
+ *
+ * Ported from Go: https://github.com/golang/go/blob/go1.12.5/src/os/file.go#L389
```
Comment (ry) [inline os/mod.ts]: Add "Needs permissions to access env (--allow-env)."

Comment (ekaragodin) [reply os/mod.ts]: Done.

**Comment by axetroy** [comment]
it looks superfluous for me.

Isn't it better to implement in `Deno.homedir()`?

**Comment by ry** [comment]
@axetroy I was thinking that too. We would be able to avoid the env access by adding an op.

**Comment by ekaragodin** [comment]
@ry I don't have enough experience with rust, but I tried https://github.com/denoland/deno/pull/2578

**Pull request merged**